lib/dry/validation/result.rb

```diff
@@ -101,8 +101,16 @@ def failure?
 
       # Check if values include an error for the provided key
       #
-      # @api private
+      # @api public
```

Comment:
not sure if that's alright but imo would be great to have it for public consumption :)

Reply:
Yes with this PR we can make it a public API. It was my intention from the beginning, actually.